### PR TITLE
cake-6072 | Add link to global footer model

### PR DESCRIPTION
**Pull request by cschucode**
Add link to Cortex RPG in Global Footer.

### CODE REVIEW

File: includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php [hunk 1]
```diff
@@ -268,6 +268,15 @@ public function getData() {
...
 						'tracking_label' => 'explore.dnd-beyond',
 					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'text',
+							'value' => 'Cortex RPG'
+						],
+						'href' => 'https://www.cortexrpg.com/',
+						'tracking_label' => 'explore.dnd-beyond',
```
Comment (mklucsarits) [inline includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php]: Needs updating

Comment (cschucode) [reply includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php]: good catch. copy/paste and then proof read. 😉

File: includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php [hunk 2]
```diff
@@ -268,6 +268,15 @@ public function getData() {
 						'href' => 'https://www.dndbeyond.com/',
 						'tracking_label' => 'explore.dnd-beyond',
 					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'text',
+							'value' => 'Cortex RPG'
+						],
+						'href' => 'https://www.cortexrpg.com/',
```
Comment (mklucsarits) [inline includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php:277]: You can exclude the `www.` -- it always redirects to cortexrpg.com anyway.

**Review by mklucsarits — APPROVED** [review]

File: includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php (hunk 2 above)
Comment (cschucode) [reply includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php:277]: Any harm in leaving it?

**Pull request merged**